### DISCUSSION TITLE
Use pac auth on Ms CRM on-primise

### DISCUSSION THREAD

**Discussion by farzadbyt1** [microsoft/powerplatform-build-tools#1097] [Q&A]
Hi guys.
I want to use pac auth on ms crm on-primise 
my odata url is : https://192.168.95.132/BayatSoft/api/data/v9.1
but pac try to connect : https://192.168.95.132/api/data/v9

I run the bellow command:
pac auth create --environment https://192.168.95.132/BayatSoft/api/data/v9.1  --username administrator --password *****
or 
pac auth create --environment https://192.168.95.132/BayatSoft   --username administrator --password *****
or 
pac auth create --environment https://192.168.95.132  --username administrator --password *****

and I try to use Url Rewrite to map https://192.168.95.132/api/data/v9 to https://192.168.95.132/BayatSoft/api/data/v9.1
but none of the ways I try working! 

this link is ok  and working: https://192.168.95.132/BayatSoft/api/data/v9.1

the error I take from pac: 

Microsoft PowerPlatform CLI
Version: 1.37.4+g82c3669
Online documentation: https://aka.ms/PowerPlatformCLI
Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions

Error: Can't resolve AAD/OAuth authority for https://192.168.95.132/api/data/v9

**Comment by MattB-msft** [comment]
I am sorry, but PAC cli is not supported for connecting to on premises deployments. 

It is coded to support only online environments.
